content/docs/command-reference/commit.md

```diff
@@ -69,13 +67,14 @@ There are many cases where the last step is not desirable (for example rapid
 iterations on an experiment). The `--no-commit` option prevents it (on the
 commands where it's available). The file hash is still computed and added to the
 `dvc.lock` or `.dvc` file, but the actual data is not cached. And this is where
-the `dvc commit` command comes into play: It performs that last step when
-needed.
+`dvc commit` comes into play: It performs that last step when needed.
 
 Note that it's best to avoid the last three scenarios. They essentially
-force-update the `dvc.lock` or `.dvc` files and save data to cache. They are
-still useful, but keep in mind that DVC can't guarantee reproducibility in those
-cases.
+force-update the metafiles and cache. They are still useful, but keep in mind
+that DVC can't guarantee reproducibility in those cases.
+
+> Note that committing data does not affect code, `dvc.yaml`, or `.dvc` files.
```

Comment:
committing data affects .dvc though ... affects is probably not the right word here

Reply:
Simplified.

I would even like to remove this note but I guess `commit` can be confused between DVC and Git and they're really nothing like each other...